jena-core/src/test/java/org/apache/jena/reasoner/test/TestRDFSReasoners.java

```diff
@@ -95,6 +95,7 @@ public static TestSuite suite() {
     /**
      * Build a single named query test
      */
+    @SuppressWarnings("unused")
     private static void constructSingleQuerytests(TestSuite suite, String manifest, String test, ReasonerFactory rf, Resource config) throws IOException {
```

Comment:
Yes, sorry. I shouldn't have included that, as well as being cosmetic (and unsuccessfully cosmetic!) it's not relevant to this test case.

Would you like me to resubmit the PR without this?